cli/cmd/root.go

```diff
@@ -133,6 +133,8 @@ func NewCmdRoot() *cobra.Command {
 	f := cmd.PersistentFlags().VarPF(telemetryLevel, "telemetry-level", "", "Telemetry level (none, errors, stats, all)")
 	f.DefValue = "all"
 
+	iUUID := cmd.PersistentFlags().VarPF(&invocationUUID, "invocation-uuid", "", "useful for when using Open Telemetry integration for tracing and logging to be able to correlate logs and traces through many services")
+	iUUID.DefValue = "<UUID>"
```

Comment:
```suggestion
	iUUID.DefValue = "<NEW-RANDOM-UUID>"
```

Maybe?

Reply:
Done in [2a843d7](https://github.com/cloudquery/cloudquery/pull/18878/commits/2a843d7ae6cdd86d7521e301e6ed82a6b384a25d)